fenix/app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -163,13 +171,21 @@ private val lightColorPalette = FirefoxColors(
     layerAccent = PhotonColors.Ink20,
     layerAccentNonOpaque = PhotonColors.Violet70A12,
     layerAccentOpaque = Color(0xFFEAE4F9),
+    layerWarning = PhotonColors.Yellow20,
+    layerConfirmation = PhotonColors.Green20,
+    layerError = PhotonColors.Red10,
+    layerInfo = PhotonColors.Blue50A44,
```

Comment:
This is gonna sound somewhat crazy, but could I request that you reorder the parameters to follow the ordering in the [figma stickersheet](https://www.figma.com/file/pEyGeE4KV5ytYHeXMfLcEr/Mobile-Styles?type=design&node-id=1637-23964&mode=design&t=IjKu9IYFyEcnzFRv-0)?

My rationale is that I go up and down the stickersheet to just make sure everything is named correctly, has the right color, description, etc. Having the ordering thrown off when comparing our code with the figma would just make that comparison and check process that much harder.

Reply:
Specially, these layer variables need to go after `gradientEnd`